worlds/mlss/__init__.py

```diff
@@ -111,6 +116,8 @@ def create_items(self) -> None:
         for item in itemList:
             if item.classification != ItemClassification.filler and item.classification != ItemClassification.skip_balancing:
                 freq = item_frequencies.get(item.itemName, 1)
+                if item.itemName == "Beanstar Emblem":
+                    freq = (0 if self.options.goal != "emblem_hunt" else self.options.emblems_amount)
```

Comment:
Probably good to just be safe here and get the actual number
```suggestion
                    freq = (0 if self.options.goal != "emblem_hunt" else self.options.emblems_amount.value)
```